### DISCUSSION TITLE
dosya nasıl eklenir?

### DISCUSSION THREAD

**Discussion by Mert190mert2015** [community/community#68145] [GitHub Education]
### Hi everyone,

github da dosya nasıl eklenir

**Comment by samus-aran** [comment]
Hi @Mert190mert2015 ! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
